{"name": "#正确的账号、错误的密码、正确的验证码", "status": "passed", "description": "错误的账号", "attachments": [{"name": "log", "source": "4e618710-65ae-4150-a818-ae7d09f10a91-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "fc9a431d-4446-4ab1-a61c-21d1e9adb7d9-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "094c7ae4-e60b-4e6a-bc1a-00a64d20d4ed-attachment.txt", "type": "text/plain"}], "start": 1618374224310, "stop": 1618374228851, "uuid": "d8b2b92d-484e-4855-aef8-d32d8647fd44", "historyId": "883a541f97fe6c8c9ddfd4c2f0cae9b6", "testCaseId": "60c800e0ac0825f582650298f219264b", "fullName": "script.test_login.Test_login#test_error_account", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "story", "value": "用例2：正确的账号、错误的密码、正确的验证码"}, {"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_login"}, {"name": "host", "value": "DESKTOP-BN9QCCJ"}, {"name": "thread", "value": "12728-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_login"}], "links": [{"type": "test_case", "url": "https://mail.qq.com/", "name": "qq邮箱"}]}